logic/dapp.js

```diff
@@ -85,7 +85,7 @@ DApp.prototype.verify = function(transaction, sender, cb, tx) {
 	}
 
 	const amount = new Bignum(transaction.amount);
-	if (!amount.isEqualTo(0)) {
+	if (amount.isGreaterThan(0)) {
```

Comment:
why is it changing the logic here?

Reply:
This change comes from development. I've just solved the conflicts after the merge following the existing pattern. The commit was introduced in this PR https://github.com/LiskHQ/lisk/pull/2432
I was checking with @ManuGowda one of the reviewers, if there was an important change of the logic overlooked. He confirmed me @nazarhussain was aware of this possibility and everything is alright.